doc/lightning-fundchannel_start.7.md

```diff
@@ -43,7 +43,7 @@ RETURN VALUE
 
 [comment]: # (GENERATE-FROM-SCHEMA-START)
 On success, an object is returned, containing:
-- **funding_address** (string): The address to send funding to for the channel
+- **funding_address** (string): The address to send funding to for the channel. DO NOT SEND COINS TO THIS ADDRESS YET.
```

Comment:
looks like it is missing a `make doc-all` to add the warning propriety to the doc, and from the CI it looks like we need to add the `warning_usage` also to the fundchannel.schema.json and rerun `make doc-all`